### PR TITLE
Update st2

### PR DESCRIPTION
**Pull request by Aratz**
This updates the st2-docker submodule to the latest version of st2

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-export ST2_VERSION=3.2.0
+export ST2_VERSION=3.7.0
```
Comment (matrulda) [inline .env:3]: Is this the most recent version available? It would be ideal to use 3.8.

**Review by matrulda — COMMENTED** [review]
Nice, left a comment you can have a look at. And another general one, where you able to install snpseq_packs with these changes?

File: .env (hunk 1 above)
Comment (Aratz) [reply .env:3]: haha good point, it was actually the latest version when I made these changes, I'll update this

**Comment by Aratz** [comment]
Yes I was able to run the docker container locally with these changes, `st2-docker` is only used to test snpseq_packs locally

**Review by matrulda — APPROVED** [review]
Great!

**Pull request merged**